src/env.js

```diff
@@ -9,8 +9,12 @@ export const env = createEnv({
   server: {
     POSTGRES_URL: z.string().url(),
     CLERK_SECRET_KEY: z.string(),
-    FLOWCORE_DATACORE: z.string().url(),
+    FLOWCORE_BASEURL: z.string(),
+    FLOWCORE_WEBHOOK_BASEURL: z.string(),
```

Comment:
I appreciate no default values! ❤️ They always tend to cause trouble in the long run